client/core/trade.go

```diff
@@ -1406,9 +1409,8 @@ func (t *trackedTrade) checkRedemptionFeeConfirms(match *matchTracker) bool {
 		// Confirmed will be set in the db.
 		return true
 	}
-	if match.Side == order.Maker {
-		return match.Status >= order.MakerRedeemed
-	}
+	// Checking taker redemption fees might be a bit early, but there will be
+	// no more ticks after the status is set to MatchConfirmed.
 	return match.Status >= order.MatchComplete
```

Comment:
It's not clear what the intention is here. For maker, the difference between `MakerRedeemed` and `MatchComplete` is just a successful `sendRedeemAsync`, which should come immediately after the status is set to `MakerRedeemed` anyway, and seems unrelated to whether or not we want to check confirmations.

Reply:
Yeah you're right, this change is unnecessary. For the redemption I don't think there's anything we can do due to the irregular confirmation updates I mentioned https://github.com/decred/dcrdex/pull/2062#discussion_r1083591489 . If one updates takes us from 0 to 10, and the status is set to `MatchConfirmed`, `tick` will no longer run. Unless we update `matchIsActive` to take into account these fees.

Reply:
I believe this is fine since unlike with the swap transactions, there should be no delays after the maker broadcasts their txn.